obspy/io/mseed/tests/test_mseed_special_issues.py

```diff
@@ -1176,13 +1179,45 @@ def test_read_file_with_empty_blocks_in_between(self):
         reference = _read_mseed(filename)
         del reference[0].stats.mseed
 
+        # Fill with zero bytes.
         for length in (128, 256, 512, 1024, 2048, 4096, 8192):
             with io.BytesIO() as buf:
                 buf.write(rec1)
                 buf.write(b'\x00' * length)
                 buf.write(rec2)
                 buf.seek(0, 0)
-                st = _read_mseed(buf)
+                # This will raise 1 warning per 128 bytes.
+                with warnings.catch_warnings(record=True) as w:
+                    warnings.simplefilter("always")
+                    st = _read_mseed(buf)
+                self.assertEqual(len(w), length // 128)
+
+            # Also explicitly test the first warning message which should be
+            # identical for all cases.
+            self.assertEqual(
+                w[0].message.args[0],
+                "readMSEEDBuffer(): Not a SEED record. Will skip bytes "
+                "4096 to 4223.")
+
+            # Remove things like file-size and what not.
+            del st[0].stats.mseed
+            self.assertEqual(reference, st)
+
+        # Try the same thing but fill with random bytes.
+        # The seed is not really needed but hopefully guards against the
+        # very very rare case of random bytes making up a valid SEED record.
```

Comment:
Haha 😆 reminds me of this: https://en.wikipedia.org/wiki/Infinite_monkey_theorem